README.md

```diff
@@ -788,6 +799,88 @@ To disable this behavior, you can set the following config value in your ini fil
     ckanext.dcat.translate_keys = False
 
 
+## Structured data
+
+To add [structured data](https://developers.google.com/search/docs/guides/intro-structured-data) to dataset pages, activate the `structured_data` plugin in your ini file.
```

Comment:
I took me quite a while to discover why I wasn't seeing the structured data json-ld after activating the `structured_data` plugin. Here we should mention that you need in `structured_data` in addition to the `dcat` plugin.

This basically means that if someone wants the schema.org support they also need to enable the links to RDF representation for datasets and catalog, as these are automatically added by the `dcat` plugin. This is probably fine, but we might want to add a config option to disable this for people that just want the schema.org support.